config/routes.rb

```diff
@@ -10,7 +10,7 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
 
-  delete '/logout', to: 'sessions#destroy'
+  get '/logout', to: 'sessions#destroy'
```

Comment:
Oh that answers my question from above. Shouldn't we keep it as a delete since it is going to a destroy method so it's RESTful?